shell/creators/pkg/init

```diff
@@ -3,6 +3,9 @@
 const fs = require('fs-extra');
 const path = require('path');
 const https = require('https');
+const semver = require('semver');
+
+const UI_EXTENSIONS_API_VERSION = '1.2.0';
```

Comment:
Probably a bad idea to have this in two places.

I think this will be the same version as the version in `shell/package.json` - so reading it from there in both places in this PR would be preferable.